httptools/src/main/java/org/pesmypetcare/httptools/HttpParameter.java

```diff
@@ -0,0 +1,143 @@
+package org.pesmypetcare.httptools;
+
+import androidx.annotation.NonNull;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.util.Objects;
+
+/**
+ * @author Santiago Del Rey
+ */
+public class HttpParameter {
+    private String name;
+    private String value;
+
+    /**
+     * Default constructor.
+     */
+    private HttpParameter() {
+    }
+
+    /**
+     * Creates an http parameter with a string as value
+     *
+     * @param name The parameter name
+     * @param value The parameter value
+     */
+    public HttpParameter(String name, String value) {
+        this.name = name;
+        this.value = value;
+    }
+
+    /**
+     * Creates an http parameter with an integer as value
+     *
+     * @param name The parameter name
+     * @param value The parameter value
+     */
+    public HttpParameter(String name, int value) {
+        this.name = name;
+        this.value = String.valueOf(value);
+    }
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Libraries/pullRequest?prid=5538337)